code/datums/uplink/gear loadout.dm

```diff
@@ -1,6 +1,6 @@
 /datum/uplink_item/item/gear_loadout
 	category = /datum/uplink_category/gear_loadout
-	item_cost = 70
+	item_cost = 20
```

Comment:
The price must be tweaked so a team of 4 can not buy more than one loadout crate.
Or you need to implement a system that limits the quantity of loadout creates that can be bought globally (to 1)